### PR TITLE
Wsp/formplayer submit

### PR DESCRIPTION
**Pull request by wpride**
Switch to submitting forms from Formplayer rather than from the browser

PRed into wsp/django-auth, hold off on merge until that's merged

cross https://github.com/dimagi/formplayer/pull/52

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js [hunk 1]
```diff
@@ -79,36 +79,10 @@ FormplayerFrontend.reqres.setHandler('startForm', function (data) {
...
-                data.onerror({message: message});
-                // TODO change submit button text to something other than
-                // "Submitting..." and prevent "All changes saved!" message
-                // banner at top of the form.
-            },
-        });
+        FormplayerFrontend.request("clearForm");
+        // TODO form linking
+        FormplayerFrontend.trigger("apps:list");
+        showSuccess(gettext("Form successfully saved"), $("#cloudcare-notifications"), 2500);
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: so this shows success regardless of whether the form submission was successful?

File: corehq/apps/registration/static/registration/js/new_user.ko.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+                                        callback(result.isValid);
+                                    },
+                                }
+                            );
+                        } else if (self.email() !== undefined) {
+                            _private.resetEmailFeedback(false);
+                        }
+                    },
+                    message: django.gettext("There is already a user with this email."),
+                }
```
Comment (dimagimon) [inline corehq/apps/registration/static/registration/js/new_user.ko.js]: Missing trailing comma. (comma-dangle)

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js (hunk 1 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js]: fixed, thanks.

File: corehq/apps/app_manager/views/formdesigner.py [hunk 3]
```diff
@@ -115,6 +116,10 @@ def _form_too_large(app, form):
             if getattr(f, 'schedule', False) and f.schedule.enabled
         ])
 
+    if tours.VELLUM_CASE_MANAGEMENT.is_enabled(request.user) \
+        and app.vellum_case_management and form.requires_case():
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/formdesigner.py]: E125 continuation line with same indent as next logical line

File: corehq/pillows/xform.py [hunk 4]
```diff
@@ -2,30 +2,31 @@
...
+    XFormDuplicate, SubmissionErrorLog
 from pillowtop.checkpoints.manager import PillowCheckpoint, PillowCheckpointEventHandler
-from pillowtop.es_utils import ElasticsearchIndexInfo, get_index_info_from_pillow
 from pillowtop.pillow.interface import ConstructedPillow
 from pillowtop.processors.elastic import ElasticProcessor
 from pillowtop.processors.form import AppFormSubmissionTrackerProcessor
-from pillowtop.reindexer.change_providers.couch import CouchViewChangeProvider
-from pillowtop.reindexer.reindexer import get_default_reindexer_for_elastic_pillow, \
-    ElasticPillowReindexer, ResumableElasticPillowReindexer
+from pillowtop.reindexer.reindexer import ElasticPillowReindexer, ResumableBulkElasticPillowReindexer
```
Comment (dimagimon) [inline corehq/pillows/xform.py]: F401 'ElasticPillowReindexer' imported but unused

**Comment by benrudolph** [comment]
i think the branch you're PRing into needs a master merge

**Comment by wpride** [comment]
Will recreate into master once that's merged

**Comment by wpride** [comment]
close for PR into master https://github.com/dimagi/commcare-hq/pull/12402

**Pull request closed without merging**